Source/WebKit/NetworkProcess/curl/NetworkDataTaskCurl.h

```diff
@@ -93,24 +94,28 @@ class NetworkDataTaskCurl final : public NetworkDataTask, public WebCore::CurlRe
     void blockCookies();
     void unblockCookies();
 
+    void updateNetworkLoadMetrics(WebCore::NetworkLoadMetrics&);
+
     String suggestedFilename() const override;
     void deleteDownloadFile();
 
+    WebCore::FrameIdentifier m_frameID;
+    WebCore::PageIdentifier m_pageID;
+    WebCore::ShouldRelaxThirdPartyCookieBlocking m_shouldRelaxThirdPartyCookieBlocking { WebCore::ShouldRelaxThirdPartyCookieBlocking::No };
+    RefPtr<WebCore::SecurityOrigin> m_sourceOrigin;
+
     State m_state { State::Suspended };
 
     RefPtr<WebCore::CurlRequest> m_curlRequest;
     WebCore::ResourceResponse m_response;
     unsigned m_redirectCount { 0 };
     unsigned m_authFailureCount { 0 };
 
-    WebCore::FrameIdentifier m_frameID;
-    WebCore::PageIdentifier m_pageID;
-
     FileSystem::PlatformFileHandle m_downloadDestinationFile { FileSystem::invalidPlatformFileHandle };
 
     bool m_blockingCookies { false };
 
-    WebCore::ShouldRelaxThirdPartyCookieBlocking m_shouldRelaxThirdPartyCookieBlocking { WebCore::ShouldRelaxThirdPartyCookieBlocking::No };
+    bool m_failsTAOCheck { false };
```

Comment:
I'm wondering why only curl port adds `m_failsTAOCheck`? Cocoa and Soup ports are using `m_networkLoadMetrics.failsTAOCheck`.

Reply:
The answer is similar to the previous question. Curl port does not store NetworkLoadMetrics like Cocoa and SOUP. Therefor, curl port adds `m_failsTAOCheck` to store results of failsTAOCheck.